### PR TITLE
!str GraphInterpreterSpecKit must not extend AkkaSpec

### PR DESCRIPTION
**Pull request by rkuhn**

### CODE REVIEW

**Comment by rkuhn** [comment]
@drewhk does this bring back the benchmark results you are used to?

File: akka-stream-tests/src/test/scala/akka/stream/impl/fusing/GraphInterpreterSpecKit.scala [hunk 1]
```diff
@@ -11,8 +11,9 @@ import akka.stream.stage.{ GraphStage, GraphStageLogic, InHandler, OutHandler, _
 import akka.stream.testkit.AkkaSpec
 import akka.stream.testkit.Utils.TE
 import akka.stream.impl.fusing.GraphInterpreter.GraphAssembly
+import akka.event.NoLogging
 
-trait GraphInterpreterSpecKit extends AkkaSpec {
```
Comment (ktoso) [inline akka-stream-tests/src/test/scala/akka/stream/impl/fusing/GraphInterpreterSpecKit.scala]: great catch.

**Comment by drewhk** [comment]
Failure is unrelated: https://github.com/akka/akka/issues/17784

**Comment by drewhk** [comment]
fixes #https://github.com/akka/akka/issues/18844

**Pull request merged**